### PR TITLE
Fixed routes to breaking changes in auto_routes package

### PR DESCRIPTION
**Pull request by sebastianbuechler**
Since version 0.6.0 the auto_route package requires a list instead of class fields for routes declaration.
See: [https://stackoverflow.com/questions/62853598/flutter-auto-route-generator-not-building-formatexception-not-an-instance-of-l](https://stackoverflow.com/questions/62853598/flutter-auto-route-generator-not-building-formatexception-not-an-instance-of-l)

Not sure if the newly generated routes.gr.dart file needs to be included or not.

### CODE REVIEW

File: packages/stacked_services/example/lib/app/routes.dart [hunk 1]
```diff
@@ -7,12 +7,22 @@ import '../ui/views/second_screen.dart';
...
-    transitionsBuilder: TransitionsBuilders.zoomIn, durationInMilliseconds: 400)
-class $Router {
-  @initial
-  HomeScreen homeScreenRoute;
-
-  FirstScreen firstScreenRoute;
-
-  SecondScreen secondScreenRoute;
-}
+    routes: <AutoRoute>[
```
Comment (RodolfoSilva) [inline packages/stacked_services/example/lib/app/routes.dart]: Hey @sebastianbuechler great job.
Just a small change, can you adjust the indentation in the `routes` parameter?

**Review by RodolfoSilva — CHANGES_REQUESTED** [review]

File: packages/stacked_services/example/lib/app/routes.dart (hunk 1 above)
Comment (sebastianbuechler) [reply packages/stacked_services/example/lib/app/routes.dart]: Sure, done.

**Review by RodolfoSilva — APPROVED** [review]
LGTM!

**Comment by FilledStacks** [comment]
Awesome. Thanks

**Pull request merged**